README.md

```diff
@@ -247,6 +242,16 @@ The following options are available:
 - `managed` - Derives the wallet from the hash of the dAPI name (or data feed ID). This means the wallet derivation is
   agnostic to update parameters, and the same wallet is used when the dAPI is upgraded/downgraded.
 
+#### `stage`
+
+An identifier of the deployment stage. This is used to distinguish between different deployments, for example `dev`,
+`prod-1` or `prod-2`. The stage value can have 256 characters at maximum and can only include lowercase alphanumeric
+characters and hyphens.
+
+#### `version`
```

Comment:
Just a heads-up, we are using `version` on `signed-api`, `nodeVersion` on `airnode-feed`. At some point, it'd be good to be consistent on this.

Reply:
We might, but I don't think it's that big of a deal. Personally I prefer using `version`, but `nodeVersion` comes from Airnode, where it was named to distinguish it with `ois.version` and `ois.oisFormat`. I think for other services where there is no ambiguity it's OK to just use `version`.